source/cloud-security/amazon/services/supported-services/security-hub.rst

```diff
@@ -0,0 +1,140 @@
+
+Storing Security Hub events in S3 buckets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is achieved by integrating Security Hub with EventBridge as it is carried out, for example, for the :ref:`Amazon WAF integration <amazon_waf>`. 
+
+Types of Security Hub integration with EventBridge
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Security Hub can automatically send all new findings and all updates to existing findings to EventBridge as *Security Hub Findings - Imported* events, each one containing a single finding. 
+- Security Hub sends findings that are associated with custom actions to EventBridge as *Security Hub Findings - Custom Action* events when these are triggered.
+- To be able to process the Security Hub Insights, custom actions can also be used to send sets of insight results to EventBridge as *Security Hub Insight Results* events. Insight results are the resources that match an insight.
+
+   .. note::
+      To get events of the last two types to EventBridge, it is required to create the custom action in Security Hub. Please refer to the `documentation <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-custom-actions.html>_` related to said topic to achieve this.
+
+Each type of event contains a `specific format <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-event-formats.html>`_ from which the Wazuh integration takes every relevant **detail** field and value along with the **detail-type** value.
+
+- *Security Hub Findings - Imported*
+
+   - ``findings``: list which contains, in JSON format, the finding that is sent by the event. Each event sends a single finding.
+
+- *Security Hub Findings - Custom Action*
+
+   - ``actionName``: Name of the action.
+   - ``actionDescription``: Description of the action.
+   - ``findings``: list which contains, in JSON format, the finding that is sent by the event. Each event sends a single finding.
+
+- *Security Hub Insight Results*
+   
+   - ``actionName``: Name of the action.
+   - ``actionDescription``: Description of the action.
+   - ``insightArn``: ARN of the insight.
+   - ``insightName``: Name of the insight.
+   - ``resultType``: Resource Id.
+   - ``number of results``: Number of results, maximum of 100.
+   - ``insightResults``: list of the results from the insight.
+
+More information on how to configure each type in the `AWS Security Hub related section <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-integration-types.html>`_.
+
+Amazon Simple Queue Service and Amazon S3 bucket event notifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The SQS and S3 configurations required are the same as the ones for the :ref:`Custom Logs Buckets integration <amazon_custom_logs_configuration>`.
```

Comment:
The other services' documentation repeats the configuration steps over an over again, so I wouldn't mind if you include this part instead of referencing the other service.

I think we should find a solution to centralize all services configuration to avoid repetition, but that is not the scope of this issue.